web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -459,12 +457,31 @@ describe("profile - shared", () => {
             businessPersona: persona as BusinessPersona,
           }),
         });
-        renderPage({ business });
+        renderPage({ business, isAuthenticated: IsAuthenticated.TRUE });
         expect(
           screen.queryByText(Config.profileDefaults.default.noteForBusinessesFormedOutsideNavigator)
         ).not.toBeInTheDocument();
       }
     );
+
+    it.each(businessPersonas)(
+      "shows Note alert for %s business that set DateOfFormation but did NOT pay",
+      (persona) => {
+        const business = generateBusinessForProfile({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
```

Comment:
pebble: is it necessary to test `OWNING` / would iterating over`nonOwningPersonas` make more sense? They would never be asked to pay for formation. What is the scenario your trying to test for Owning, maybe another test is needed?